code/modules/religion/burdened/psyker.dm

```diff
@@ -258,36 +288,47 @@
 	spell_max_level = 1
 	invocation_type = INVOCATION_NONE
 	spell_requirements = SPELL_REQUIRES_NO_ANTIMAGIC
-	channel_message = span_notice("You focus on your weapon...")
+	channel_message = span_notice("You focus on your trigger fingers...")
 	charge_overlay_icon = 'icons/effects/effects.dmi'
 	charge_overlay_state = "purplesparkles"
-	channel_flags = IGNORE_HELD_ITEM
+	channel_time = 5 SECONDS
+	channel_flags = IGNORE_USER_LOC_CHANGE
```

Comment:
does puling out your gun interrupt this now?

Reply:
yes